app/models/foreman_google/gce.rb

```diff
@@ -60,6 +60,7 @@ def disks
     # This should return interface compatible with Fog::Server
     # implemented by ForemanGoogle::Compute
     def find_vm_by_uuid(uuid)
```

Comment:
If I pass `uuid` as integer it will fail:
```
Passing integer instead of the string fails with:
Google::Protobuf::TypeError: Invalid argument for string field 'instance' (given Integer).
```
Maybe would be worth to have `identity: uuid.to_s` as a safe check

Reply:
why not, but the uuid should be string as it is string in the db, but sure, rather be safe :+1: